wiki/Beatmap/Pattern/osu!mania/Chordstream/en.md

```diff
@@ -0,0 +1,21 @@
+---
+stub: true
+---
+
+# Chordstream
+
+*For stream types specific to 4-key, refer to [Stream](/wiki/Beatmap/Pattern/osu!mania/Stream).*
+
+**Chordstreams** are a type of [stream](/wiki/Beatmap/Pattern/osu!mania/Stream) which involve different chord sizes. The stream is usually on 1/4 with chords every 1/1 or 1/2 snap interval. In key modes outside of 4K, "chordstream" usually refers to streams involving 3 or more notes within the chords.
+
+![](img/chordstream.png)
+
+**Chordstreams** may also include **double streams**, which are two streams occurring simultaneously. These are more common in 7K osu!mania or higher key modes.
```

Comment:
"the 7K" doesn't seem right here... I think this would be better:
```suggestion
**Chordstreams** may also include **double streams**, which are two streams occurring simultaneously. These are more common within 7K and higher key modes of osu!mania.
```